README.md

```diff
@@ -114,36 +114,37 @@ For more usage details, see our detailed [documentation](#documentation).
+| `(a)`                      | 6.60x                | 0.84x                 | 5.70x            | 13.84x             |
+| `(a, b)`                   | 5.02x                | 0.72x                 | 4.15x            | 10.53x             |
+| `(a:)`                     | 9.83x                | 0.83x                 | 5.94x            | 17.24x             |
+| `(a:, b:)`                 | 7.66x                | 0.70x                 | 4.66x            | 16.24x             |
+| `(a, b:)`                  | 7.87x                | 0.70x                 | 4.64x            | 13.38x             |
+| `(a, *args)`               | 2.08x                | 0.78x                 | 1.57x            | 3.58x              |
+| `(a:, **kwargs)`           | 2.46x                | 0.59x                 | 1.78x            | 6.84x              |
+| `(a, *args, b:, **kwargs)` | 1.67x                | 0.67x                 | 1.12x            | 4.10x              |
 
 \* `Dry::Core`
 [may cause incorrect behavior caused by hash collisions](https://github.com/dry-rb/dry-core/issues/63).
 
 Results using Ruby 2.7.8 (because these gems raise errors in Ruby 3.x):
 
-|Method arguments|`DDMemoize` (1.0.0)|`Memoist` (0.16.2)|`Memoized` (1.1.1)|`Memoizer` (1.0.3)|
-|--|--|--|--|--|
-|`()` (none)|22.57x|2.27x|23.46x|2.63x|
-|`(a)`|20.96x|14.29x|20.54x|11.97x|
-|`(a, b)`|18.22x|13.21x|17.76x|11.34x|
-|`(a:)`|30.66x|23.52x|25.37x|21.61x|
-|`(a:, b:)`|27.31x|21.98x|23.02x|20.31x|
-|`(a, b:)`|26.21x|20.85x|21.57x|19.20x|
-|`(a, *args)`|3.06x|2.23x|3.10x|1.92x|
-|`(a:, **kwargs)`|2.67x|2.18x|2.39x|2.02x|
-|`(a, *args, b:, **kwargs)`|2.14x|1.80x|1.89x|1.70x|
+| Method arguments           | `ddmemoize` (1.0.0) | `memoist` (0.16.2) | `memoized` (1.1.1) | `memoizer` (1.0.3) |
+|----------------------------|---------------------|--------------------|--------------------|--------------------|
+| `()` (none)                | 19.77x              | 2.41x              | 21.53x             | 3.08x              |
+| `(a)`                      | 16.41x              | 11.80x             | 17.03x             | 10.38x             |
+| `(a, b)`                   | 15.09x              | 11.17x             | 15.21x             | 9.96x              |
+| `(a:)`                     | 20.91x              | 17.19x             | 18.48x             | 15.66x             |
+| `(a:, b:)`                 | 20.48x              | 17.30x             | 18.74x             | 16.18x             |
+| `(a, b:)`                  | 19.17x              | 16.03x             | 16.93x             | 15.57x             |
+| `(a, *args)`               | 2.91x               | 2.08x              | 2.90x              | 1.81x              |
+| `(a:, **kwargs)`           | 2.61x               | 2.20x              | 2.61x              | 2.18x              |
+| `(a, *args, b:, **kwargs)` | 2.13x               | 1.79x              | 1.99x              | 1.71x              |
```

Comment:
Markdown tables are intended to look good and be readable in plaintext!  My IDE (RubyMine) does this automatically.

Reply:
Cool! I'm totally happy to have these nicely formatted, but would you be willing to change our benchmarks code so that it spits out a formatted table like this? Happy to help if that doesn't make sense.

Reply:
That will take me a while. My IDE does it for me automatically, so I have never needed code to do it.

Reply:
Perhaps to get this merged quicker I should revert to no spaces?

Reply:
Done!